Sources/dep/get.swift

```diff
@@ -101,6 +101,9 @@ extension Sandbox: Fetcher {
     func fetch(url url: String) throws -> Fetchable {
         let dstdir = Path.join(prefix, Package.name(forURL: url))
         if let repo = Git.Repo(root: dstdir) where repo.origin == url {
+            if !repo.hasVersion {
```

Comment:
This is needed below the if too. So I think maybe instead put it in the RawClone initializer and make it `throws`.